src/celeritas/ext/detail/ScopedRootErrorHandler.hh

```diff
@@ -0,0 +1,45 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2022 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/ext/detail/RootErrorHandler.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+namespace celeritas
+{
+namespace detail
+{
+
+//---------------------------------------------------------------------------//
+/*!
+ *  ROOT Error Handler for Celeritas.
+ *
+ */
+
+void RootErrorHandler(int rootlevel, bool abort_bool, const char *location,
+                          const char *msg);
```

Comment:
It shouldn't be necessary to publicly declare this function right? It could just remain in the C++ file.

Reply:
Correct.  I had kept it in the header to use it as the pattern for the signature.